udata/settings.py

```diff
@@ -166,7 +166,7 @@ class Defaults(object):
 
     LINKCHECKING_ENABLED = True
     LINKCHECKING_IGNORE_DOMAINS = []
-    LINKCHECKING_CACHE_DURATION = 60 * 5  # in seconds
+    LINKCHECKING_MIN_CACHE_DURATION = 60 * 5  # in seconds
```

Comment:
Maybe it make more sense to express the duration in minutes (ie. checking multiple times by minutes does not seems realistic)

Reply:
I agree. But it won't be backward compatible. Do we have a doctrine on that?

Reply:
I changed the variable name anyway, so not really backward compatible from the beginning ;-) I'll make the change.